src/Altinn.App.PlatformServices/Implementation/AppResourcesSI.cs

```diff
@@ -437,5 +431,24 @@ private byte[] ReadFileContentsFromLegalPath(string legalPath, string filePath)
 
             return null;
         }
+        
+        /// <inheritdoc />
+        [Obsolete("GetOptions method is obsolete and will be removed in the future. Use the corresponding method in IAppOptions interface instead.", false)]
```

Comment:
If we are planning a new major version with breaking changes. Should we consider changing the error param so it generates compile error instead of a warning?

Reply:
Good point, went one step further and removed the method since we are creating a new major version.